framework/material/Form.kt

```diff
@@ -491,16 +497,21 @@ object Form {
     fun MultilineTextInput(
         state: MultilineTextInputState = rememberMultilineTextInputState(),
         value: TextFieldValue,
-        modifier: Modifier,
+        modifier: Modifier = Modifier,
+        textFieldPadding: Dp = 0.dp,
         icon: Icon? = null,
         focusReq: FocusRequester = remember { FocusRequester() },
         onValueChange: (TextFieldValue) -> Unit,
-        onTextLayout: (TextLayoutResult) -> Unit
+        onTextLayout: (TextLayoutResult) -> Unit,
+        border: Border? = null,
```

Comment:
Let's refactor this out and just modify `modifier` from the outside - it's more Composey.

Reply:
Done. `TextInput` also uses this pattern, but we expect it to have a border almost everywhere it's used rather than the case for `MultilineTextInput` where we expect it not to have one by default.

Reply:
Right, I didn't know we were already using this pattern in `TextInput`. This is highly relevant!

In `TextInput` we check if the input is **enabled** or not, and if not, we replace the border colour with a faded colour. Hence the need for a dedicated `border` property.

We don't have any disabled `MultilineTextInput`s yet, so it seems reasonable to keep the code as is.